src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String colorOfBall = colorSupplier.getRandomColor();
+        Random random = new Random();
+        int numberOfBall = random.nextInt(101);
+        return new Ball(colorOfBall, numberOfBall);
```

Comment:
same, let's make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method. And lets make number 101 as constant field https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names